sagify/sagemaker/sagemaker.py

```diff
@@ -18,7 +18,7 @@
 class SageMakerClient(object):
     def __init__(self, aws_profile, aws_region, aws_role=None, external_id=None):
 
-        if aws_role and external_id:
```

Comment:
No need for these to be tied; one may only need to use a role as per the description on the PR.